src/Rule/Email.php

```diff
@@ -30,61 +29,51 @@ final class Email implements RuleWithOptionsInterface, SkipOnErrorInterface, Whe
     use SkipOnErrorTrait;
```

Comment:
Description:

> Validates that the value is a valid email address.

But rule does not perform the actual validation, it's done by handler.